src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -985,6 +985,7 @@ public boolean contains(String id) {
      *
      * @param updateHandlerUri The Update Handler URI, in the format: <code>designDoc/update1</code>
      * @param docId            The document id to update.
+     *                         If null, a POST request is invoked.
```

Comment:
Saying "a POST request is invoked" relies on someone knowing that a POST will do a create. I think it is better to say that if no ID is provided a document will be be created.